src/content/docs/accounts/accounts-billing/account-setup/choose-your-data-center.mdx

```diff
@@ -45,26 +45,17 @@ New Relic offers almost all the same active products, features, support offering
 * [Log patterns](/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns/#availability)
 * Deprecated products and features are not available.
 
-[New Relic's incident intelligence](/docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence) service operates solely in the US whether you store your data in New Relic’s US region data center or New Relic’s EU region data center, by using New Relic incident intelligence, you consent that New Relic may move and store your data in the US region.
+[New Relic's incident intelligence](/docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence) service operates solely in the US whether you store your data in New Relic's US region data center or New Relic's EU region data center, by using New Relic incident intelligence, you consent that New Relic may move and store your data in the US region.
 
-## Regions and account hierarchy [#global-account-hierarchy]
+## Accounts in two regions [#account-structure]
 
-You can create accounts in each region. If your data is currently being hosted in the US region, you must create a new account to store data in the EU region. You cannot view EU data from a US account, or US data from an EU account.
+When you sign up for New Relic, a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#organization-accounts) is created. This is what contains all your accounts and data. When creating an organization, you must specify the data center region. 
 
-The data collected remains separate. The data cannot be aggregated or migrated between accounts.
+By default, a New Relic organization and its accounts can only be in a single data center region. However, some customers with [partnership account structures](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partnership-accounts-users-subscriptions#partnerships) can have accounts in both regions. Some details for customers with accounts in both regions: 
```

Comment:
Is it worth mentioning how to get this set up if they are interested?

Reply:
usually yeah, but we don't want to encourage them in this case. good question